src/Input/Silk.NET.Input.Common/Hat.cs

```diff
@@ -5,11 +5,26 @@
 
 namespace Silk.NET.Input.Common
 {
+    /// <summary>
+    /// Represents a joystick hat, a D-Pad.
```

Comment:
A hat is not a DPad.

![image](https://user-images.githubusercontent.com/11160611/61156823-49dd7c80-a4ec-11e9-89d5-c02dad4a5a71.png)

This is the hat:
![image](https://user-images.githubusercontent.com/11160611/61156842-58c42f00-a4ec-11e9-9752-0a033f4dfc91.png)

Reply:
Most of the controllers I've seen map the d-pads as hats, I guess it's like a fruit/apples thing. What should the documentation say instead there? I'm not really sure how to explain what a hat is if it's not just a d-pad.

Reply:
Just remove all mentions of a DPad in the Hat documentation ;)